processing-service-container/src/test/java/com/github/cafdataprocessing/processing/service/tests/RepositoryConfigIT.java

```diff
@@ -374,6 +350,33 @@ public void testScope() throws ApiException
         }
 
     }
+    
+    @Test(description = "Test that 404 is returned if the key, tenantId and repositoryId are all non-existing")
```

Comment:
I think this one should be split into 3 tests, each one checking that you get a 404 if one of the parameter values do not exist while the others do exist in each test. Therefore 3 tests:
- key not exist, tenantId and repositoryId do exist
- tenantId not exist, key and repositoryId do exist
- repositoryId not exist, key and tenantId do exist